{{cookiecutter.project_slug}}/{{cookiecutter.python_package}}/{{cookiecutter.blok_name}}/tests/test_model.py

```diff
@@ -1,10 +1,11 @@
-from anyblok.tests.testcase import BlokTestCase
+from anyblok.conftest import *  # noqa: F401,F403
```

Comment:
I'm not sure how to avoid importing ``*`` or what would be the prefered ways ?

If I do something likes
```python
from anyblok.conftest import rollback_registry, init_session
```
I get following lint errors:
```
./tests/test_model.py:1:1: F401 'anyblok.conftest.rollback_registry' imported but unused
./tests/test_model.py:1:1: F401 'anyblok.conftest.init_session' imported but unused
./tests/test_model.py:7:35: F811 redefinition of unused 'rollback_registry' from line 1
```